source/features/label-color-picker.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+// FIXME: whenever color palette changes its value, the label color example above the edit form doesn't change it's value
```

Comment:
But when the save button is pressed, the label color changes.

Reply:
I think you'll have to use the `insert-text-textarea` module instead of `.value = .value` in `labelDelegateEvent`